### ISSUE TITLE
Example stats

### ISSUE THREAD

**Issue by Phlya** [mirnylab/distiller-nf#96]
Hi guys,

Would be great to add some examples of stats for good and bad experiments with explanations which step in the protocol might have failed, and how to understand this. I am currently troubleshooting why my recent Hi-C have not been working well, and with the coded pair type annotation this part of it is a little more complicated than I expected.

Also, more of a pairsamtools issue, but related: W pair type is still called C there in the docs (it's the same thing, right? I seem to remember it mentioned at some point).

**Comment by Phlya** [comment]
Also... As we briefly discussed with Sergey and Johan, havingE fragment-level stats like in hiclib and other pipelines (e.g. dangling ends, self-circles etc) would be very helpful for troubleshooting failed experiments.

**Comment by sergpolly** [comment]
Let's collect all of the `stats` update requests in one place.
So far we have this:
https://github.com/mirnylab/pairsamtools/issues/59
https://github.com/mirnylab/pairsamtools/issues/56
https://github.com/mirnylab/pairsamtools/issues/54
https://github.com/mirnylab/pairsamtools/issues/5
https://github.com/mirnylab/distiller-nf/issues/94
https://github.com/mirnylab/distiller-nf/issues/90

Please, @golobor , @Phlya , @nvictus  review the list, prioritize and let's go from there

**Comment by gfudenberg** [comment]
I couldn't find these in the referenced posts, but it would also be nice to have:
a) P(s) for different read orientations separately as well-- this is useful for for finding where they converge and reads can be interpreted as "just measuring contact frequency"
b) number of reads with mitochondria is a nice stat (mito_vs_anyReads, mito_vs_mito, etc.)
c) number of single-sided and double-sided read pairs

**Comment by Phlya** [comment]
I guess it should be possible to address @gfudenberg 's point (a) quite easily, since these counts are all already present in the output of stats - https://github.com/mirnylab/pairsamtools/issues/68. Although, perhaps, the bins can be optimized a bit to make more smooth curves?

But is having plots in the output in the plans? As an html/pdf report with different things, or just a folder with individual pngs/pdfs? Should their generation be part of stats, or a separate job, which just takes the output of stats?

**Comment by golobor** [comment]
I know that DCIC was working QC for pairs for a while, the results are
here: https://github.com/4dn-dcic/pairsqc

**Comment by Phlya** [comment]
Yeah, I've seen that and even tried to install ones without success. But considering there is `pairsamtools stats` already which calculate so many things, I don't think there is any point in using another QC tool?
